pkgs/development/compilers/dotnet/combine-packages.nix

```diff
@@ -56,5 +56,9 @@ mkWrapper "sdk" (buildEnv {
     );
   };
 
-  inherit (cli) meta;
+  meta = {
+    description = "${cli.meta.description} (combined)";
+    maintainers = with lib.maintainers; [ corngood ];
```

Comment:
Maintainers here should be inherited or a combination of all maintainers from all combined packages imo